README.md

```diff
@@ -128,8 +128,17 @@ In general, a policy is defined by:
 - *actor*: The user, group or role, that is linked to the policy
 - *action*: The action allowed on this resource (e.g. acl:Read for GET requests)
 - *resource*: The urn of the resource being targeted (e.g. urn:entity:x)
-- *resource_type*: The type of the resource (e.g. entity, entity_type,
-  subscription,  device, ...)
+- *resource_type*: The type of the resource.
+
+The currently supported resource types for the policies are:
```

Comment:
in general? i think this is only for the authorization rules in place, correct? in which case i would specify that this can be - of course - customized by creating proper OPA policies + envoy filters